<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TodoClass</class>
 <widget class="QMainWindow" name="TodoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Todo</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Todo.qrc">
    <normaloff>:/Todo/appicon.png</normaloff>:/Todo/appicon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow { 
	background-color: rgb(24, 24, 24);
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">
QScrollBar:vertical {              
        border: 1px solid #999999;
        background:white;
        width:4px;    
        margin: 0px 0px 0px 0px;
    }
    QScrollBar::handle:vertical {
        background: black;
        min-height: 0px;
    }
    QScrollBar::add-line:vertical {
        background: black;
        height: 0px;
        subcontrol-position: bottom;
        subcontrol-origin: margin;
    }
    QScrollBar::sub-line:vertical {
        background: black;
        subcontrol-position: top;
        subcontrol-origin: margin;
    }</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="inputTaskArea" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget#inputTaskArea{
	background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(24, 24, 24, 255), stop:0.386364 rgba(67, 67, 67, 255));
}</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="2" column="1">
        <widget class="QPushButton" name="upbtn">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>8</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>5</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
border-radius: 6px;
border: 1px solid  rgb(179, 179, 179);
background-color: rgb(34, 34, 34);
color: white;
}

QPushButton:hover {
	border-color: rgb(225, 225, 225);
}</string>
         </property>
         <property name="text">
          <string>^</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="taskAdderLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPlainTextEdit" name="todoText">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="font">
            <font>
             <strikeout>false</strikeout>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPlainTextEdit{
border: 2px solid rgb(24, 24, 24);
background-color: rgb(255, 255, 255);
width:3px;    
border-radius: 8px;
}</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>2</number>
           </property>
           <item>
            <widget class="QPushButton" name="addText">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>20</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>6</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
border-radius: 6px;
border: 1px solid  rgb(179, 179, 179);
background-color: rgb(34, 34, 34);
color: white;
padding: 2px 2px 2px 2px;
opacity: 0.6;
}

QPushButton:hover {
	border-color: rgb(225, 225, 225);
}</string>
             </property>
             <property name="text">
              <string>Add Task</string>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="reloadTasks">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>20</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>6</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
border-radius: 6px;
border: 1px solid  rgb(179, 179, 179);
background-color: rgb(34, 34, 34);
color: white;
padding: 2px 2px 2px 2px;
}

QPushButton:hover {
	border-color: rgb(225, 225, 225);
}</string>
             </property>
             <property name="text">
              <string>Reload Tasks</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="styleSheet">
       <string notr="true">QWidget#scrollArea {
	background-color: transparent;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>390</width>
         <height>533</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget#scrollAreaWidgetContents {
	background-color: transparent;
}
</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Todo.qrc"/>
 </resources>
 <connections/>
</ui>
